specification.markdown

```diff
@@ -173,21 +173,65 @@ An EBML Stream is a file that consists of one or many EBML Documents that are co
 
 ## Elements semantic
 
-### Element Template
+### EBML Schema
 
-Element Name:
+An EBML Schema is an XML Document that defines the properties, arrangement, and usage of EBML Elements that compose a specific EBML Document Type. The relationship of an EBML Schema to an EBML Document may be considered analogous to the relationship of an XML Schema to an XML Document. An EBML Schema MUST be clearly associated with one or many EBML Document Types. An EBML Schema must be expressed as well-formed XML. An EBML Document Type is identified by a unique string stored within the EBML Header element called DocType; for example `matroska` or `webm`.
```

Comment:
I added the links at https://github.com/MediaArea/ebml-specification/commit/e706a1db60b6163ca2c4e86818a0a941b9b6f09a.
I suggest re-formatting to RFC style formatting of references and links be done when it's closer to done. For now I added an issue for this at https://github.com/Matroska-Org/ebml-specification/issues/41